### PR TITLE
feat: create get_by_id service and controller

### PR DESCRIPTION
**Pull request by EduardoHACampos**

### CODE REVIEW

File: backend/src/controllers/cars.controller.ts [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 import { updateCarsService } from "../services/cars/updateCars.service"
 import { Response, Request } from "express";
 
+const getByIdCarsController = async (req: Request, res: Response): Promise<Response> => {
+
+    const carId: number = Number(req.params.id)
+
+    const car:TCarResponse= await getByIdCarsService(carId)
+
+    return res.json(car)
```
Comment (luccaseabracruz) [inline backend/src/controllers/cars.controller.ts]: missing .status(200)

File: backend/src/routes/cars.routes.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import { Router } from "express";
-import { deleteCarController, updateCarController } from "../controllers/cars.controller";
+import { deleteCarController, getByIdCarsController, updateCarController } from "../controllers/cars.controller";
 
 const carRoutes = Router()
 
 carRoutes.post("",)
 carRoutes.get("",)
+carRoutes.get(":id",getByIdCarsController)
```
Comment (luccaseabracruz) [inline backend/src/routes/cars.routes.ts]: Some endpoints have "/" and others don't. Can you review it?

**Review by luccaseabracruz — CHANGES_REQUESTED** [review]
The requested changes were commented in the specific code.

File: backend/src/controllers/cars.controller.ts (hunk 1 above)
Comment (EduardoHACampos) [reply backend/src/controllers/cars.controller.ts]: Added the missing .status(200) as suggested.

File: backend/src/routes/cars.routes.ts (hunk 2 above)
Comment (EduardoHACampos) [reply backend/src/routes/cars.routes.ts]: I've corrected the inconsistency mentioned. Thank you

**Review by luccaseabracruz — APPROVED** [review]

**Pull request merged**